toolhub/integrations/rapidapi/loader.py

```diff
@@ -8,6 +8,7 @@
 # TODO: use package resources.
 _PARENT_DIR = os.path.realpath(os.path.join(os.getcwd(), os.path.dirname(__file__)))
 FUNCTIONS_FILE = os.path.join(_PARENT_DIR, "functions.json")
+SENDGRID_FILE = os.path.join(_PARENT_DIR, "sendgrid.json")
```

Comment:
Why should a general interface reference a specific integration?
If we'd like to add custom category to function mappings, we could just introduce a `collections.json`.

Reply:
I'll delete this for now